builder/core/data.py

```diff
@@ -178,7 +178,7 @@ class PKG_TOOLS(Enum):
         'pkg_update': 'zypper refresh && zypper --non-interactive patch',
         'pkg_install': 'zypper install -y',
         'variables': {
-            'python': "python3.9",
+            'python': "python3",
```

Comment:
on `opensuse/leap:15.3` that we're currently using, `python3` == python3.6

maybe we should just be updating the docker image